deploy/scripts/src/test-accountManagement.ts

```diff
@@ -407,13 +436,33 @@ export function changeEmail (): void {
         const endTime = Date.now()
 
         check(res, {
-          'is status 200': r => r.status === 200,
-          'verify page content': r => (r.body as string).includes('Your services')
+          'status is 200': (r) => r.status === 200,
```

Comment:
```suggestion
          'is status 200': r => r.status === 200,
```
Keep the `check` naming convention consistent